mllib/src/main/scala/org/apache/spark/ml/feature/PolynomialExpansion.scala

```diff
@@ -38,9 +38,12 @@ class PolynomialExpansion extends UnaryTransformer[Vector, Vector, PolynomialExp
 
   /**
    * The polynomial degree to expand, which should be larger than 1.
+   * Default: 2
    * @group param
    */
-  val degree = new IntParam(this, "degree", "the polynomial degree to expand")
+  val degree = new IntParam(this, "degree", "the polynomial degree to expand",
+    ParamValidate.gt[Int](2))
```

Comment:
`gt(1)`. Users may set degree to `1` to turn off polynomial expansion in tuning.